__tests__/jubilation-horizontal-bar/jubilation-horizontal-bar-test.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { defaultContext } from '../../src/helpers/context';
+import HorizontalBar from '../../src/jubilation-horizontal-bar';
+
+
+describe('JubilationHorizontalBarChart', () => {
+  it('should have a default data prop of []', () => {
+    expect(shallow(<HorizontalBar />).instance().props.data).toEqual([]);
+  });
+
+  it('should have a default gapWidth prop of 0.05', () => {
+    expect(shallow(<HorizontalBar />).instance().props.gapWidth).toBe(0.05);
+  });
+
+  it('should set a uuid attribute', () => {
```

Comment:
Are you trying to test the uniqueness?  The presence? As written, this test feels like it's asserting format, which I don't think is really what you're going for.